packages/yourloops/components/menus/hcp-team-scope-menu.tsx

```diff
@@ -0,0 +1,217 @@
+import Typography from '@mui/material/Typography'
+import MenuLayout from '../../layout/menu-layout'
+import MenuItem from '@mui/material/MenuItem'
+import ListItemIcon from '@mui/material/ListItemIcon'
+import ListItemText from '@mui/material/ListItemText'
+import Divider from '@mui/material/Divider'
+import AddIcon from '@mui/icons-material/Add'
+import TeamEditDialog from '../../pages/hcp/team-edit-dialog'
+
+const classes = makeStyles()((theme: Theme) => ({
+  sectionTitle: {
+    fontWeight: 'bold',
+    paddingLeft: theme.spacing(2),
+    paddingBottom: theme.spacing(1)
+  },
+  menu: {
+    paddingTop: theme.spacing(1),
+    paddingBottom: theme.spacing(1)
+  }
+}))
+
+export const HcpTeamScopeMenu: FunctionComponent = () => {
+  const { t } = useTranslation('yourloops')
+  const {
+    classes: {
+      sectionTitle,
+      menu
+    }
+  } = classes()
+  const { getMedicalTeams, getPrivateTeam } = useTeam()
+  const { selectTeam, selectedTeamId } = useSelectedTeamContext()
+  const { createTeam } = useTeam()
+  const alert = useAlert()
+  const navigate = useNavigate()
+  const [teamCreationDialogData, setTeamCreationDialogData] = React.useState<TeamEditModalContentProps | null>(null)
+  const theme = useTheme()
+  const isMobile: boolean = useMediaQuery(theme.breakpoints.only('xs'))
+
+  const privateTeam = getPrivateTeam()
+  const sortedMedicalTeams = TeamUtils.sortTeamsByName(getMedicalTeams())
+  const hasMedicalTeams = sortedMedicalTeams.length > 0
+  const availableTeams = [privateTeam, ...sortedMedicalTeams]
+
+  const selectedTeam = availableTeams.find((team: Team) => team.id === selectedTeamId)
+  const isSelectedTeamPrivate = selectedTeam.type === TeamType.private
+  const selectedTeamName = isSelectedTeamPrivate ? t('hcp-team-menu-private-practice-option') : selectedTeam.name
+
+  const privatePracticeIcon = <PersonIcon data-testid="private-practice-icon" />
+  const medicalTeamIcon = <GroupsOutlinedIcon data-testid="medical-team-icon" />
+  const selectedTeamIcon = isSelectedTeamPrivate ? privatePracticeIcon : medicalTeamIcon
+
+  const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null)
+  const isMenuOpen = !!anchorEl
+
+  const onSelectTeam = (teamId: string): void => {
+    if (teamId !== selectedTeamId) {
+      selectTeam(teamId)
+      navigate('/')
+    }
+    closeMenu()
+  }
+
+  const onCreateTeam = (): void => {
+    setTeamCreationDialogData({ team: null, onSaveTeam })
+    closeMenu()
+  }
+
+  const onSaveTeam = async (createdTeam: Partial<Team> | null): Promise<void> => {
+    if (createdTeam) {
+      try {
+        await createTeam(createdTeam as Team)
+        alert.success(t('team-page-success-create'))
+      } catch (reason: unknown) {
+        alert.error(t('team-page-failed-create'))
+      }
+    }
+    setTeamCreationDialogData(null)
+  }
+
+  const closeMenu = (): void => {
+    setAnchorEl(null)
+  }
+
+  return (
+    <>
+      <Box maxWidth={250}>
```

Comment:
The maxWidth of the box 😄 I can create a `MENU_MAX_WIDTH_PX` const to reuse in the menus, it would be cleaner (I kept what was already done before but you're right)